client/verta/verta/_internal_utils/_utils.py

```diff
@@ -361,7 +344,7 @@ def get_default_workspace(self):
         if workspace_id:
             return self.get_workspace_name_from_id(workspace_id)
         else:  # old backend
```

Comment:
```suggestion
        else:
```

Would you mind also removing this comment, for clarity?